test/tools/yulrun.cpp

```diff
@@ -153,7 +161,7 @@ Allowed options)",
 		else
 			input = readUntilEnd(cin);
 
-		interpret(input);
+		interpret(input, arguments.count("interactive"), arguments.count("enable-external-calls"));
```

Comment:
maybe "interactive" and "enable-external-calls" could be extracted into a variable and used here and within the options some lines above.